Networking/Networking/Model/CouponReport.swift

```diff
@@ -0,0 +1,46 @@
+import Codegen
+import Foundation
+
+public struct CouponReport {
+    /// ID of the coupon
+    public let couponId: Int64
```

Comment:
nit: I think we generally use `*ID` in the codebase, maybe we can keep this undocumented convention?

```suggestion
    public let couponID: Int64
```